.travis.yml

```diff
@@ -0,0 +1,62 @@
+language: cpp
+dist: focal
+
+matrix:
+  include:
+    - os: linux
+      addons:
+        apt:
+          packages:
+            - cmake
+            - gcc-10
+            - g++-10
+            - libstdc++-10-dev
+            - libz-dev
+            - uuid-dev
+      env:
+        - ENV_SETUP="CC=gcc-10 && CXX=g++-10"
+        - BEFORE_INSTALL=""
+        - RUN_TESTS="sudo update-alternatives --install /usr/bin/g++ g++ /usr/bin/g++-10 100 && sudo update-alternatives --install /usr/bin/gcc gcc /usr/bin/gcc-10 100 && chmod +x libVC_test libVC_testUtils && ./libVC_testUtils && ./libVC_test "
+    - os: linux
+      addons:
+        apt:
+          sources:
+            - llvm-toolchain-focal-15
+            - sourceline: "deb http://apt.llvm.org/focal/ llvm-toolchain-focal-15 main"
+              key_url: "https://apt.llvm.org/llvm-snapshot.gpg.key"
+          packages:
+            - cmake
+            - gcc-10
+            - g++-10
+            - libstdc++-10-dev
+            - libz-dev
+            - uuid-dev
+            - clang-15
+            - lldb-15
+            - lld-15
+            - clangd-15
+            - clang-format-15
+            - clang-tools-15
+            - llvm-15-dev
+            - lld-15
+            - lldb-15
+            - llvm-15-tools
+            - libomp-15-dev
+            - libc++-15-dev
+            - libc++abi-15-dev
+            - libclang-common-15-dev
+            - libclang-15-dev
+            - libclang-cpp15-dev
+            - libunwind-15-dev
+      env:
+        - ENV_SETUP="CC=clang-15 && CXX=clang-15"
+        - LLVM_CONFIG="llvm-config-15"
+        - BEFORE_INSTALL="sudo update-alternatives --install /usr/bin/clang++ clang++ /usr/bin/clang++-15 100 && sudo update-alternatives --install /usr/bin/clang clang /usr/bin/clang-15 100 && sudo update-alternatives --install /usr/bin/opt opt /usr/bin/opt-15 100"
+        - RUN_TESTS="sudo update-alternatives --install /usr/bin/g++ g++ /usr/bin/g++-10 100 && sudo update-alternatives --install /usr/bin/gcc gcc /usr/bin/gcc-10 100 && chmod +x libVC_test libVC_testJit libVC_testUtils && ./libVC_testUtils && ./libVC_test && ./libVC_testJit "
+
+before_install:
+  - eval "${ENV_SETUP}" && eval "${BEFORE_INSTALL}"
+
+
+script:
+  - mkdir build && cd build && cmake .. && make 
```

Comment:
uhm . . .
Is this a side effect of the [git fire](https://juststickers.in/wp-content/uploads/2017/04/in-case-of-fire.png) procedure?

Reply:
No no, it is for Travis CI https://app.travis-ci.com/github/Focshole/libVersioningCompiler/builds/256694521

Reply:
woops